### PR TITLE
Add callback transformer

### PR DESCRIPTION
**Pull request by kdebisschop**
Resolves issue #37

### CODE REVIEW

File: src/Transformers/Callback.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     * Transformer columns.
+     *
+     * @var string[]
+     */
+    protected $columns = [];
+
+    /**
+     * The replace function.
+     *
+     * @var string
```
Comment (ecourtial) [inline src/Transformers/Callback.php:28]: Isn't it supposed to be a callable ?

**Review by ecourtial — CHANGES_REQUESTED** [review]
A small question

File: src/Transformers/Callback.php (hunk 1 above)
Comment (kdebisschop) [reply src/Transformers/Callback.php]: Spot on. Fix pushed. Thanks.

**Review by ecourtial — APPROVED** [review]

**Review by AurelienRasselet — APPROVED** [review]

**Review by Nicolas-Masson-Wizaplace — APPROVED** [review]

**Comment by kdebisschop** [comment]
Hey all -- I may have resolved the signed commit issue (I did not realize I was missing that aspect)

I suspect, however, if I was to for instance reword the last commit and force-push the same code, it would trigger a new review requirement. But I could do that if it would help on your end. LMK if I should, otherwise I will leave as-is.

**Comment by ecourtial** [comment]
@kdebisschop Not an issue with the signed commit :) We will consider that later (for all the contributors), if really needed or not.
We merge this PR. 
I would like to wait to have the full support for PHP8 before tagging the new version.

**Pull request merged**